.github/workflows/build.yml

```diff
@@ -0,0 +1,134 @@
+name: Binary build
+
+on:
+  push:
+    branches:
+      - 'grpc@1.24.x'
+  pull_request:
+    branches:
+      - 'grpc@1.24.x'
+
+jobs:
+  all_node_glibc:
+    name: All Simple Native Builds
+    strategy:
+      matrix:
+        node: ['4', '5', '6', '7', '8', '9', '10', '11', '12', '13']

```

Comment:
Isn't 14 available already?

Reply:
I wanted to have this PR be about getting the builds we currently have on master working on this branch with this workflow, and then another PR making updates, including building for new versions.